src/Tools.h

```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include <string>
+
+void StartBotProcess(const std::string& CommandLine);
+
+void SleepFor(int seconds);
+
+void KillSc2Process(unsigned long pid);
+
+bool MoveReplayFile(const char* lpExistingFileName, const char* lpNewFileName);
```

Comment:
Now MoveReplayFile is implemented but where is it used?

Reply:
Its used in the LadderManager.cpp where there was a previous call to MoveFile, this should be fine

Reply:
I mean that the new function is not used instead of old MoveFile. At least I don't see it in the diff.

Reply:
Yes you're right - I'll fix this.